### ISSUE TITLE
Accessibility review

### ISSUE THREAD

**Issue by EstiShay** [codeforpdx/tenantfirstaid#80]
1. Create accessibility checks PR template
2. Screen reader testing - currently failing with Voiceover and Safari
3. Keyboard navigation
4. Magnification changes, window resizing
5. Changes announced - currently failing
6. Contrast checks
7. Page structure

This issue will probably be broken into sub-issues as I get underway.
